src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task {
+
+    public Deadline(String description) {
+        super(description);
+    }
+
+    @Override
+    public String getType(){
+        return ("D");
```

Comment:
perhaps use a private variable to store the type information and return it instead? This way, it's easier to change the type's letter in one place rather than ctrl-F and searching throughout the code